src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -146,18 +146,18 @@
   --#{$menu-toggle}--m-plain--disabled--BackgroundColor: transparent; // picking icon color rather than text...?
 
   // Typeahead
-  --#{$menu-toggle}--m-typeahead--c-text-input-group__utilities--c-button--PaddingInlineEnd: var(--pf-t--global--spacer--sm);
+  --#{$menu-toggle}--m-typeahead--c-text-input-group__utilities--c-button--PaddingInlineEnd: var(--pf-t--global--spacer--control--horizontal--plain);
   --#{$menu-toggle}--m-typeahead__button--AlignSelf: stretch;
 
   // * Menu toggle small
-  --#{$menu-toggle}--m-small--PaddingBlockStart: var(--pf-t--global--spacer--xs);
-  --#{$menu-toggle}--m-small--PaddingBlockEnd: var(--pf-t--global--spacer--xs);
+  --#{$menu-toggle}--m-small--PaddingBlockStart: var(--pf-t--global--spacer--control--vertical--compact);
+  --#{$menu-toggle}--m-small--PaddingBlockEnd: var(--pf-t--global--spacer--control--vertical--compact);
   --#{$menu-toggle}--m-small--ColumnGap: calc(var(--pf-t--global--spacer--control--horizontal--compact) / 2);
   --#{$menu-toggle}--m-small--FontSize: var(--pf-t--global--font--size--body--sm);
   --#{$menu-toggle}--m-small__controls--MinWidth: calc(var(--#{$menu-toggle}--FontSize) + var(--pf-t--global--spacer--xs) + var(--pf-t--global--spacer--xs));
 
   // Status icon
-  --#{$menu-toggle}__status-icon--MarginInlineEnd: var(--pf-t--global--spacer--md);
+  --#{$menu-toggle}__status-icon--MarginInlineEnd: var(--pf-t--global--spacer--gap--text-to-element--default);
```

Comment:
Have you confirmed this with design? I wouldn't think this is the right use of text-to-element... I'd probably just use the small spacer since it's spacing itself from the toggle icon? If we were to use text-to-element here, I'd expect to use this in the form control, too.

Reply:
TODO: Update to non-gap spacer for now, re-evaluate in gap token spacer issue